src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegion.java

```diff
@@ -466,7 +467,7 @@ public SimpleInterval getReadSpanLoc() {
      * @param padding the padding, in BP, we want to add to either side of this active region extended region
      * @return a non-null array of bytes holding the reference bases in referenceReader
      */
-    public byte[] getAssemblyRegionReference( final IndexedFastaSequenceFile referenceReader, final int padding ) {
+    public byte[] getAssemblyRegionReference(final ReferenceSequenceFile referenceReader, final int padding ) {
```

Comment:
name of the `referenceReader` parameter seems weird given that the type is called `ReferenceSequenceFile`